test-requires.txt

```diff
@@ -1,6 +1,3 @@
 psycopg2==2.7.4
-pytest
-mock
-docutils<0.13  # 0.13 + py27 cause 'setup.py check' error. see: https://bugs.python.org/issue23063
-flake8
--e.
+pytest==3.6.2
```

Comment:
- Never upgraded when use cache and disable upgrade option
- Manage with dependency update service like pyup.io